example/pubspec.yaml

```diff
@@ -22,3 +21,7 @@ dev_dependencies:
 dependency_overrides:
   w_flux:
     path: ../
+  react:
+    git:
+      url: https://github.com/Workiva/react-dart.git
+      ref: v7_wip
```

Comment:
I'm not sure what the plan is for maintaining this. Are we ok with leaving this directed to the v7 branch for now?